src/pages/error/PageNotFound.tsx

```diff
@@ -47,8 +73,6 @@ const PageNotFound = () => {
 
     return (
         <PageContainer>
-            <Topbar isNavigationOpen={false} hideNavigationMenu />
```

Comment:
Do you want to remove the `hideNavigationMenu` prop from `Topbar`?

Reply:
No... I think we should keep it for now. It is useful and I think we could use it for certain error/loading pages where we don't want the user leaving.